pkg/api/types_test.go

```diff
@@ -373,6 +374,7 @@ func TestAdminAPIParity(t *testing.T) {
 	// TODO: why don't we just include all of these in the admin type?
 	notInAdmin := []*regexp.Regexp{
 		regexp.MustCompile(`^\.Config\.RunningUnderTest$`),
+		regexp.MustCompile(`^\.Config\.RootDockerConfig$`), // TODO: I don't feel this should be in admin API, should it?
```

Comment:
@jim-minter @kargakis Do you think we should be able to rotate this secret using AdminAPI? My feeling is that we should be able as this is secret, which has a hard external dependency, and I would like to be able to update it as fast as I can without rolling out new config or code. 

Anything external "touching" should be easy to manage. WDYT?

Reply:
I think this is fine with a restart. We will need to have this as Geneva action either way

Reply:
yes, we need a geneva action for this.